packages/button/src/index.css

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
```

Comment:
@ahmadshaheer, this is not good practice. The button should have only the rule for the font family. You can configure the Storybook app to import the font.

Fonts should be imported via the consumer app config (e.g., Storybook app or SigNoz). Here, just use the rule like “font-family: Inter”.

Reply:
@ahmadshaheer : We need to setup up common configs at the base level. 

Components should just import from there base configs

Reply:
@sharpshooter90, I had added it inside button, in case someone only uses our button package from the components library. but that doesn't seem to be the case most of the time.

I made the changes to import the font in Storybook. 

We don't need to specify `font family: Inter` in button, since I have added it inside the [tailwind.config.js](https://github.com/SigNoz/components/blob/03da38700d73a7460fe7c027b2e0c946435f593b/packages/tailwind-config/tailwind.config.js#L10-L11)

Reply:
@YounixM, I have made the changes you suggested to this [PR](https://github.com/SigNoz/components/pull/26)